zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -3347,7 +3348,42 @@ paths:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
           $ref: "#/components/responses/InternalServerError"
-
+  /usage-metrics/search:
+    post:
+      tags:
+        - Usage metrics
+      operationId: searchUsageMetrics
+      summary: Query usage metrics
+      description: |
+        Search for usage metrics by given start date and end date
```

```suggestion
        Search for usage metrics by given start date and end date.
```

:wrench: Descriptions should end in a period.

---

Adding this